### PR TITLE
feat: add devnet flag to set devnet tag

### PR DESCRIPTION
**Pull request by allnil**

### CODE REVIEW

File: wvm-apps/wvm-exexed/crates/reth-exexed/src/main.rs [hunk 1]
```diff
@@ -22,6 +23,14 @@ use serde_json::to_string;
 use types::types::ExecutionTipState;
 use wevm_borsh::block::BorshSealedBlockWithSenders;
 
+pub fn get_network_tag() -> &'static str {
+    let devnet_flag = env::var("DEVNET").unwrap_or(String::from("false")).to_lowercase();
+    if devnet_flag == "true" {
+        return "Devnet"
```
Comment (charmful0x) [inline wvm-apps/wvm-exexed/crates/reth-exexed/src/main.rs]: "Devnet v0.1.0" to align with Alphanet naming

Comment (allnil) [reply wvm-apps/wvm-exexed/crates/reth-exexed/src/main.rs]: hmmmm
or it is 0.2.0??

Comment (allnil) [reply wvm-apps/wvm-exexed/crates/reth-exexed/src/main.rs]: cause it is a next version

Comment (charmful0x) [reply wvm-apps/wvm-exexed/crates/reth-exexed/src/main.rs]: 0.2.0 sgtm!

**Pull request merged**